core/src/main/java/io/grpc/inprocess/InProcessTransport.java

```diff
@@ -360,6 +361,11 @@ public void setCompressor(Compressor compressor) {}
 
       @Override
       public void setDecompressor(Decompressor decompressor) {}
+
+      // TODO(lukasz) should we return something here?
```

Comment:
Interesting. What would the static type of the value be? If it's (for example) `SocketAddress`, would you then test whether it's an instance of `InetSocketAddress` or some custom type (e.g. `InProcessSocketAddress`)?